python/cudf/cudf/io/parquet.py

```diff
@@ -73,6 +74,9 @@ def write_to_dataset(
     partition_cols : list,
         Column names by which to partition the dataset
         Columns are partitioned in the order they are given
+    row_group_cols : list,
```

Comment:
1. The optional values in this docstring should specify their default values and optionality.

See https://pandas.pydata.org/pandas-docs/version/1.1.5/reference/api/pandas.DataFrame.to_parquet.html
> partition_cols : list, optional, default None

2. Options like `root_path` and `**kwargs` with no default values should not have a trailing comma after the type name.

3. Types labeled as `string` should be labeled as `str` instead.